src/reducers/job.ts

```diff
@@ -12,6 +12,17 @@ export interface JobState {
   jobs: Array<JobModel>;
   refreshCnt: number;
   loading: boolean;
+  jobResult?: JobResult;
+  jobResultLoading: boolean;
+}
+
+export interface JobResult {
+  resultType?: 'success' | 'error';
```

Comment:
Could we pull the values for result type into an enum in index.ts? Then we can also use the enum when returning the job result in [repository.ts](https://github.com/DataBiosphere/jade-data-repo-ui/pull/1198/files#diff-b0cc26de535914428b254ecf0a5ed0e23a1c0e9753bbcb59a2af7167f6cd37ceR613)

Reply:
Updated! :)